examples/training/classification/imagenet/basic_training.py

```diff
@@ -85,6 +85,24 @@
     "Keyword argument dictionary to pass to the constructor of the model being trained",
 )
 
+flags.DEFINE_string(
+    "use_warmup_schedule",
```

Comment:
What do you think about naming this `learning_rate_schedule` and making it a string that supports "ReduceOnPlateau" or "CosineDecayWithWarmup"?

Reply:
Good catch. Much better than having conditionals like I originally did. Updating!

Reply:
Done!